newsletter-april-2023.rst

```diff
@@ -0,0 +1,96 @@
+  The results are now manifested in the structure of the navigation sidebar and the landing page on `docs.readthedocs.io <https://docs.readthedocs.io/en/stable/>`__.
+- 📊️ All of our websites now use `Plausible <https://plausible.io/>`__ for analytics.
+- ⚙️ Added a new build variable ``READTHEDOCS_CANONICAL_URL`` that's useful for projects that want to be aware of the canonical base URL while building.
+  `View docs <https://docs.readthedocs.io/en/stable/reference/environment-variables.html#envvar-READTHEDOCS_CANONICAL_URL>`__.
+- ⚙️ Added new build variables ``READTHEDOCS_GIT_CLONE_URL``, ``READTHEDOCS_GIT_IDENTIFIER``, and ``READTHEDOCS_GIT_COMMIT_HASH`` for projects that need to access Git data while building.
+  `View docs <https://docs.readthedocs.io/en/stable/reference/environment-variables.html#envvar- READTHEDOCS_GIT_CLONE_URL>`__.
+- 🔒️ Fixed vulnerability: `Cache poisoning: serving arbitrary content on documentation sites  <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-mp38-vprc-7hf5>`__
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+.. figure:: img/screenshot-docs-diataxis-update.png
+   :alt: A screenshot of our current documentation after the refactor
+   
+   A screenshot of how our user documentation looks in April 2023.
+
+
+Upcoming features
+-----------------
+
+- 📚️ We are still doing changes in our `user documentation <https://docs.readthedocs.io/en/stable/>`__ structure and content:
+  Once the last parts of the plan are implemented,
+  we are ready to share a bigger announcement.
+- ⚡️ *A lot of work* is happening these days on big front-end features.
+  In :doc:`the last newsletter </newsletter-march-2023>`, we mentioned the new Dashboard and :doc:`the migration to about.readthedocs.com </website-migration>`.
+  What's new is that we've also started building a **generic JavaScript client** accompanied with APIs that will give additional features to *any* documentation project or static site built and served on the Read the Docs platform.
+  
+  The first proof-of-concept for the new API and JavaScript library has been test-driven.
+  It is capable of displaying a menu matching our current :doc:`flyout menu <readthedocs:flyout-menu>` and :doc:`integrated server side search <readthedocs:server-side-search/index>`.
+
+  The new library is useful for *any* documentation framework or static site generator to allow for full control of the new flyout menu, integrated search and general access to API data.
+  See the screenshot further down where the integrated search in action on the documentation framework Docusaurus.
+
+  We plan to be rolling this out gradually for Sphinx and MkDocs projects that are currently running their own set of ``build.commands``.
+  Once that happens, we'll be back with news on the blog and documentation about how you can configure it.
```

Comment:
@humitos can you have another look here? It's a longer item, but I think it's the most exciting part of the newsletter and I feel like it's exactly the moment to split it into a blog post, yet worth mentioning the progress.

You can see it here: https://readthedocs-blog--213.org.readthedocs.build/newsletter-april-2023/

Reply:
@humitos said :+1: